addon/components/data-cell.js

```diff
@@ -0,0 +1,89 @@
+import Ember from 'ember';
+import $ from 'jquery';
+
+import Component               from 'ember-component';
+import computed                from 'ember-computed';
+import get                     from 'ember-metal/get';
+import { htmlSafe, dasherize } from 'ember-string';
+import { isEmberArray }        from 'ember-array/utils';
+
+const { defineProperty } = Ember;
+
+const defaultPropertyObject = {
+  format() {
+    return Array.prototype.slice.call(arguments, 0).join(', ');
+  }
+};
+
+const STYLING_PROPERTIES = {
+  horizontalAlign: { key: 'text-align' },
+  verticalAlign:   { },
+  backgroundColor: { },
+  fontFamily:      { },
+  fontWeight:      { },
+  italic:          { },
+  fontColor:       { },
+  borderWidth:     { },
+  borderColor:     { },
+  borderStyle:     { }
+};
+
+function copy(object) {
+  return $.extend({}, object);
+}
```

Comment:
Everything from line 1-33, is copied from the data-coll file. 
1. You didn't copy it right, because the `key` from the styling properties are missing. They are needed to work.
2. When we need those things in both files, we should extract them into a separate util
3. Replace the copy for [Ember's assign()](http://emberjs.com/api/#method_assign)
4. ...